model/account.go

```diff
@@ -0,0 +1,10 @@
+package model
+
+type Account struct {
+	ID       uint   `gorm:"primary_key"`
+	User     string `gorm:"not null;unique"`
+	Password string `gorm:"not null"`
+	Sex      string `gorm:"type:char(1)"`
```

Comment:
Just use an enumeration. Also, we should move this to character information as on new clients this isn't part of the account anymore.

Also, the correct word for this is `Gender`.